### ISSUE TITLE
Need to manually depend on flowbinding-common since 1.0.0-alpha01

### ISSUE THREAD

**Issue by hameno** [ReactiveCircus/FlowBinding#99]
Since 1.0.0-alpha01 I need to manually add a dependency to flowbinding-common, otherwise I get the following error:

```
Cannot access class 'reactivecircus.flowbinding.common.InitialValueFlow'. Check your module classpath for missing or conflicting dependencies
```

**Comment by ychescale9** [comment]
Thanks for reporting! Will fix soon.

**Comment by ychescale9** [comment]
This is fixed in `1.0.0-alpha02`.
